nextroute/schema/fleet.go

```diff
@@ -0,0 +1,234 @@
+// Package schema provides the input and output schema for nextroute.
+package schema
+
+import (
+	"fmt"
+	"time"
+)
+
+// FleetInput schema.
+type FleetInput struct {
+	Defaults       *FleetDefaults  `json:"defaults,omitempty"`
+	Vehicles       []FleetVehicle  `json:"vehicles,omitempty"`
+	Stops          []FleetStop     `json:"stops,omitempty"`
+	StopGroups     [][]string      `json:"stop_groups,omitempty"`
+	AlternateStops []FleetStop     `json:"alternate_stops,omitempty"`
+	DurationGroups []DurationGroup `json:"duration_groups,omitempty"`
+}
+
+// FleetDefaults holds the fleet input default data.
+type FleetDefaults struct {
```

Comment:
```suggestion
// FleetDefaults holds the fleet input default data.
// DEPRECATION NOTICE: this part of the API is deprecated and is no longer maintained. It will be deleted soon. Please use [Defaults] instead.
type FleetDefaults struct {
```

Reply:
yep